$NetBSD: patch-revert-37d3828773e2f847bb1ed7522b0af4fb8e736fc8,v 1.2 2013/08/22 19:16:14 asau Exp $

Revert revision 37d3828773e2f847bb1ed7522b0af4fb8e736fc8
which breaks build on NetBSD.

diff --git a/contrib/sb-sprof/Makefile contrib/sb-sprof/Makefile
index 7373c72..463ae52 100644
--- contrib/sb-sprof/Makefile
+++ contrib/sb-sprof/Makefile
@@ -2,4 +2,4 @@ MODULE=sb-sprof
 include ../vanilla-module.mk
 
 test::
-	$(SBCL) --eval '(load (format nil "SYS:CONTRIB;~:@(~A~);TEST.LISP" "$(MODULE)"))' </dev/null
+	        true
